user/inc/Arduino.h

```diff
@@ -4,6 +4,7 @@
 
 #define PARTICLE_ARDUINO_COMPATIBILITY
 #include "Particle.h"
+#include "spark_wiring_arduino.h"
```

Comment:
Yep, by default now application.h will define PARTICLE_ARDUINO_COMPATIBILITY = 1 unless PARTICLE_NO_ARDUINO_COMPATIBILITY is explicitly defined as =y on the make command line.  Also Arduino.h just includes Particle.h which is an alias for application.h.